cisco_aci/datadog_checks/cisco_aci/fabric.py

```diff
@@ -130,7 +136,19 @@ def process_eth(self, node):
             if self.ndm_enabled():
                 interface_metadata = ndm.create_interface_metadata(e, node.get('address', ''), self.namespace)
                 interfaces.append(interface_metadata)
-                self.submit_interface_status_metric(interface_metadata.status, tags, hostname)
+                device_id = '{}:{}'.format(self.namespace, node.get('address', ''))
+                tags.append('{}:{}'.format(DEVICE_USER_TAGS_PREFIX, device_id))
+
+                self.submit_interface_status_metric(
+                    interface_metadata.status,
+                    tags
+                    + [
+                        "{}:{}:{}".format(
+                            INTERFACE_USER_TAGS_PREFIX, interface_metadata.device_id, str(interface_metadata.index)
+                        ),
+                    ],
+                    hostname,
+                )
             try:
                 stats = self.api.get_eth_stats(pod_id, node['id'], eth_id)
                 self.submit_fabric_metric(stats, tags, 'l1PhysIf', hostname=hostname)
```

Comment:
this is sending additional port/interface related metrics such as `cisco_aci.fabric.port.fault_counter.crit` which may need the additional interface user tags (see my QA comment!)

Reply:
updated c514a56d33e77a75a4798c934c41a1d4a47e16cf